02-household-queries/qt_summaries-gpt-3.5-turbo-instruct-k_4.json

```diff
@@ -0,0 +1,851 @@
+                "summary": "It is important to collect information about all household members, even if they are not mandatory, as it can help with troubleshooting and avoiding delays in the future."
+            },
+            "If a non-mandatory household (HH) member already receives SNAP, and the client says they share & prepare meals, what steps should be taken?": {
+                "score_sum": 0.3842315979369787,
+                "derived_questions": [
+                    "Does the client share a kitchen with his roommates?",
+                    "Do the roommates often cook and eat together in the shared kitchen?"
+                ],
+                "entire_card": "If a non-mandatory household (HH) member already receives SNAP, and the client says they share & prepare meals, what steps should be taken?\nProvide guidance and ask the following:\nDo you know if you're included on that household member's SNAP application?\nThe SNAP benefit typically includes people who live in the same household and share/prepare meals together on the same application. Is there a reason that you were not included on the same case?\nYou can either have (household member) add you to the existing SNAP case or you can apply on your own. If you apply on your own, you would only be able to use the SNAP benefit for yourself and you would likely have to provide a written statement from (household member) stating that the two SNAP benefits would be used separately.\nPlease note that the above does NOT apply to mandatory HH members (spouses residing together, etc.)",
+                "summary": "If a non-mandatory household member already receives SNAP and the client shares a kitchen with his roommates who often cook and eat together, the client may need to be included on the same SNAP application as the household member or apply on his own with a written statement from the household member stating that the SNAP benefits will be used separately."
```

Comment:
**Relevance to the question**: 🟠 Potentially relevant
- Potentially relevant if any of the clients' roommates already receive SNAP, which is not specified in the question
- The answer to the client's question could be reasoned from this summary since it indicates that roommates may need to be included on the same SNAP application if they often cook and eat together

**Correctness as a summary of the Guru card**: 🟢 100% correct

**Helpfulness**: 🟡 Neutral